WooCommerce/Classes/ViewRelated/Orders/Order Details/Subscriptions section/UILabel+SubscriptionStatus.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+import UIKit
+import Yosemite
+
+
+// MARK: - UILabel + SubscriptionStatus Methods
+//
+extension UILabel {
+
+    /// Applies the appropriate Style for a given `SubscriptionStatus`
+    ///
+    func applyStyle(for statusEnum: SubscriptionStatus) {
+        applyFootnoteStyle()
+        applyLayerSettings()
+        applyBackground(for: statusEnum)
+    }
```

Comment:
Sorry if I sound repetitive but normally we prefer to avoid the networking model with a general view like a `UILabel`. IF this is a specific style for a view it should probably live in its view model. 

This is to avoid the possibility of reusing this extension in other places where it would get challenging to change one single place in the future.

Reply:
That's fair. As a shortcut to do this UI work quickly I borrowed heavily from the `SummaryTableViewCell`, which styles the order status label like this using `UILabel+OrderStatus`. But these could certainly be part of the view model for the cell.

Reply:
Updated in 5ef2e74.